tests/ga/test_logcollector.py

```diff
@@ -211,9 +211,8 @@ def test_log_collector_parses_commands_in_manifest(self):
 diskinfo,""".format(folder_to_list, file_to_collect)
 
         with patch("azurelinuxagent.ga.logcollector.MANIFEST_NORMAL", manifest):
-            with patch('azurelinuxagent.ga.logcollector.LogCollector._initialize_telemetry'):
```

Comment:
These unit tests mocked _initialize_telemetry because it was in LogCollector init. Now that it has been removed from init, the mock is not necessary